### ISSUE TITLE
How do I get the internal Editor to work?

### ISSUE THREAD

**Issue by JPRuehmann** [wwivbbs/wwiv#1096]
Hello
My Server Runs under Kubuntu 16.04 LTS
I can Login from another Server via SyncTerm the Menus and Settings work as intended.
But if I try to post a message ore send an Email
I get the Internal Editor, can enter Text but no Commands and the only way to close is disconnect completely. I can´t find any help in the Documentation and nothing in the Settings.
How can I make it work?
Thanks

**Comment by granitepenguin** [comment]
Did you try the /HELP ?  When the internal editor starts, it displays the following:

```
Enter message now, you can use 120 lines.
Colors: ^P-0123456789
Enter /Q to quote previous message, /HELP for other editor commands.
[---=----=----=----=----=----=----=----]----=----=----=----=----=----=----=----]

/help
                              WWIV Editor Commands
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /?             Display WWIV Editor help (this screen)
    /S             Save Message
    /Q             Quote text from original message.
    /ESY           Save Message Anonymously
    /ESN           Save Message NOT Anonymously
    /ABT           Abort Entire Message (Quit)
    /CLR           Clear Message and Start Over
    /RL            Replace Last Line
    /TI            Edit Message Title
    /LI            List Message So Far
    /C:            Center Text After "/C:"

    /SU/Old/New    Replace Old Text with New
    CTRL-P+ 0 - Z  Change Color to Digit
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
```

Are you not getting that?  If not, you might be missing files.

having it in the docs is a good idea, though.

**Comment by JPRuehmann** [comment]
No, nothing happens at all.
if I enter /HELP or /help
Which files?
Thanks,

**Comment by granitepenguin** [comment]
That screen comes from editor.msg in your gfiles directory.  Since you are compiling from source, you probably didn't extract the tarball first.  look for a gfiles.zip file in the tar.gz file or the root of your WWIV install and extract it in ${WWIVDIR}/gfiles (for example, /home/wwiv/gfiles).  That should get all the helper files you are missing.

**Comment by JPRuehmann** [comment]
Thanks
That was the point.
I thought I had expanded all the files to gfiles but it seems not.

**Comment by wwiv** [comment]
I'm also an error message to display when a help screen isn't available.  That'll help too a bit in debugging this type of things.

**Comment by granitepenguin** [comment]
I'm actually curious about the specifics of your install steps.  I'd like to understand what you did to see if there are steps missing in the documentation.

**Comment by JPRuehmann** [comment]
I had cloned the sources and tried to make it, first problem. then after several days of search I found the problem there are two dependencies that are not met and had to be cloned in the respective subdirectorys after that I had the Binaries. Than I created the User copied the Files run the Script,made the systemd settings tried to init.
I must say I still not understanding much of the Settings.
The docs no real enlightment form at least.
Sorry,
